static/src/javascripts/es6/projects/common/modules/crosswords/helpers/get-intersecting-entries.js

```diff
@@ -8,7 +8,7 @@ export default (entries, focussed) => {
     const entryHasIntersectingCell = entry => {
         const cells = helpers.cellsForEntry(entry);
         const intersecting = deepIntersection(cells, focussedCells);
-        return !! intersecting.length;
+        return !!intersecting.length;
```

Comment:
i'm the opposite :)

there is a rule, there's a rule for near enough everything you can think of. currently the rule flags this. 

the space is quite an unusual style so i left it as is. we're still going to need a way to solve style dissagreements...